backend/internal/group/store/store.go

```diff
@@ -199,13 +199,75 @@ func GetGroup(id string) (model.GroupDAO, error) {
 		return model.GroupDAO{}, err
 	}
 
-	members, err := getGroupMembers(dbClient, id, logger)
+	return group, nil
+}
+
+// GetGroupMembers retrieves members of a group with pagination.
+func GetGroupMembers(groupID string, limit, offset int) ([]model.Member, error) {
+	logger := log.GetLogger().With(log.String(log.LoggerKeyComponentName, loggerComponentName))
+
+	dbClient, err := provider.NewDBProvider().GetDBClient("identity")
 	if err != nil {
-		return model.GroupDAO{}, err
+		logger.Error("Failed to get database client", log.Error(err))
```

Comment:
Shall we avoid logging at internal packages and only log from service layer? Otherwise could get this logged twise

Reply:
There are lot of places follows the same pattern. Lets fix in a separate PR.